krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
I run it on CPython. And yeah, it did occur to me that a JIT will eat these loops for breakfast, lunch and dinner (maybe even with a distinct loop variable).

These loops certainly cause a ton of CPU on CPython. I'm half tempted to make krun quit if it's run on PyPy. What do you think?